File: server/dotnet/sdk-dotnet-example/Business/DTOs/CreatePaymentLinks/CreatePaymentLinkResponseDto.cs
using Business.Domain.Common.Enums;
using Business.Domain.Enums.PaymentLinks;

namespace Business.DTOs.CreatePaymentLinks;

public class CreatePaymentLinkResponseDto
{
    public string? PaymentLinkId { get; init; }
    
    public string? RedirectUrl { get; init; }
    
    public Status? Status { get; init; }
    
    public decimal? Amount { get; init; }
    
    public Currency? Currency { get; init; }
}

File: server/nodejs/src/business/dtos/payment-link/create-payment-link-response-dto.ts
import { Status } from '../../domain/common/enums/status';
import { Currency } from '../../domain/common/enums/currency';

export type CreatePaymentLinkResponseDto = {
    paymentLinkId?: string | null;
    redirectUrl?: string | null;
    status?: Status | null;
    amount?: number | null;
    currency?: Currency | null;
};
